app/models/orchestration_template.rb

```diff
@@ -153,7 +153,7 @@ def md5=(_md5)
   end
 
   def self.calc_md5(text)
-    Digest::MD5.hexdigest(text) if text
+    Rails.application.config.digest_class.hexdigest(text) if text
```

Comment:
@bzwei yeah I assume whatever we pick would require a data migration.

@woot4moo Once a database has an md5 in it, you can't change it without a migration or by carrying some code that handles old and new types (which is nasty and I wouldn't want).  It's possible we could leave it configurable but only allow changing once on creating a new database, and then not allow changing it again afterwards.  Seems like a hassle for customers to have to remember/maintain though, especially if we can just have a better OOtB default.